### PR TITLE
Replace `interface{}` with `any`

### PR DESCRIPTION
**Pull request by qwerty287**
like golang: https://github.com/golang/go/commit/2580d0e08d5e9f979b943758d3c49877fb2324cb

### CODE REVIEW

**Review by anbraten — APPROVED** [review]

File: pipeline/rpc/proto/woodpecker.pb.go [hunk 1]
```diff
@@ -1241,7 +1241,7 @@ func file_woodpecker_proto_rawDescGZIP() []byte {
 }
 
 var file_woodpecker_proto_msgTypes = make([]protoimpl.MessageInfo, 20)
-var file_woodpecker_proto_goTypes = []interface{}{
```
Comment (6543) [inline pipeline/rpc/proto/woodpecker.pb.go:1244]: dont touch generated files

File: pipeline/rpc/proto/woodpecker_grpc.pb.go [hunk 2]
```diff
@@ -239,7 +239,7 @@ func RegisterWoodpeckerServer(s grpc.ServiceRegistrar, srv WoodpeckerServer) {
 	s.RegisterService(&Woodpecker_ServiceDesc, srv)
 }
 
-func _Woodpecker_Version_Handler(srv interface{}, ctx context.Context, dec func(interface{}) error, interceptor grpc.UnaryServerInterceptor) (interface{}, error) {
+func _Woodpecker_Version_Handler(srv any, ctx context.Context, dec func(any) error, interceptor grpc.UnaryServerInterceptor) (any, error) {
```
Comment (6543) [inline pipeline/rpc/proto/woodpecker_grpc.pb.go:242]: same

File: server/store/mocks/store.go [hunk 3]
```diff
@@ -238,11 +238,11 @@ func (_m *Store) ConfigsForPipeline(pipelineID int64) ([]*model.Config, error) {
 
 // CreatePipeline provides a mock function with given fields: _a0, _a1
 func (_m *Store) CreatePipeline(_a0 *model.Pipeline, _a1 ...*model.Step) error {
```
Comment (6543) [inline server/store/mocks/store.go:240]: same

**Review by 6543 — CHANGES_REQUESTED** [review]
as comments

**Review by lafriks — APPROVED** [review]

**Review by 6543 — APPROVED** [review]

**Pull request merged**